README.md

```diff
@@ -1060,6 +1064,100 @@ A targeted notification response will be returned as an XML representation, prov
 </targetedNotificationResponse>
 ```
 
+### Stored File
+
+A stored file represents a file which is associated with a node and which is stored in AWS S3.
+
+#### XML Representation
+
+##### Single Supported File Usage
+
+``` xml
+<supportedFileUsage name="SOME_FILE_USAGE">
+    <fileTypes>
```

Comment:
Why is it `fileTypes` instead of `fileTypeList` like everything else?

Reply:
@rmanthorpe Because I hadn't realised that was the convention, and when I did, I only changed `supportedFileUsages` to `supportedFileUsageList` and forgot about this one. Thanks for pointing it out!